CHANGELOG.md

```diff
@@ -157,7 +157,7 @@
 - \[\#248\] Implement clojerl.Subvec and clojerl.Vector.Seq [\#737](https://github.com/clojerl/clojerl/pull/737) ([jfacorro](https://github.com/jfacorro))
 - \[\#284\] Rename the callback functions for marker protocols and remove their implementations [\#736](https://github.com/clojerl/clojerl/pull/736) ([jfacorro](https://github.com/jfacorro))
 - \[\#248\] Reducers [\#735](https://github.com/clojerl/clojerl/pull/735) ([jfacorro](https://github.com/jfacorro))
-- \[\#732\] Fix bug when checking for overriden vars [\#733](https://github.com/clojerl/clojerl/pull/733) ([jfacorro](https://github.com/jfacorro))
+- \[\#732\] Fix bug when checking for overridden vars [\#733](https://github.com/clojerl/clojerl/pull/733) ([jfacorro](https://github.com/jfacorro))
```

Comment:
This file is autogenerated from commit messages (were the typo was made), so sadly these fixes will comes back whenever the file is re-generated because of a new release.